### PR TITLE
fix(@clayui/toolbar): spritemap should be optional

### PR DESCRIPTION
**Pull request by bryceosterhaus**
fixes https://github.com/liferay/clay/issues/4139

saw the issue and was an easy fix so thought I would send over

### CODE REVIEW

File: packages/clay-pagination-bar/src/PaginationBarWithBasicItems.tsx [hunk 1]
```diff
@@ -113,95 +113,96 @@ const DEFAULT_LABELS = {
...
-
-		if (!href) {
-			item.onClick = () => {
-				if (onDeltaChange) {
-					onDeltaChange(label as number);
-				}
-			};
+export const ClayPaginationBarWithBasicItems: React.FunctionComponent<IProps> =
+	({
+		activeDelta,
```
Comment (matuzalemsteles) [inline packages/clay-pagination-bar/src/PaginationBarWithBasicItems.tsx:118]: This is just format, right? I don't see any other changes here

**Review by matuzalemsteles — APPROVED** [review]
LGTM, thanks @bryceosterhaus

**Comment by bryceosterhaus** [comment]
@matuzalemsteles my bad. I think I had a different version of prettier in my editor and it formatted on save and I didn't take a look before sending since I only added `?`.

I sent an update

**Comment by matuzalemsteles** [comment]
Thank you @bryceosterhaus, don't worry!

**Pull request merged**